Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp

```diff
@@ -130,12 +148,57 @@ void setup(py::object bpy_context)
                 if (match_func("to_mesh_clear")) BObject_to_mesh_clear = func;
             }
         }
+        else if (match_type("ObjectModifiers")) {
+            each_func{
+                if (match_func("clear")) BObject_modifiers_clear = func;
+            }
+        }        
         else if (match_type("Mesh")) {
             BMesh::s_type = type;
             each_func {
                 if (match_func("calc_normals_split")) BMesh_calc_normals_split = func;
+                if (match_func("update")) BMesh_update = func;
+                if (match_func("clear_geometry")) BMesh_clear_geometry = func;
+            }
+        }
+        else if (match_type("MeshVertices")) {
+            each_func{
+                if (match_func("add")) BMesh_add_vertices = func;
```

Comment:
Maybe we should try to make the function naming consistent so that python function name matches the end of the C++ function ?
In this example, this would be `BMesh_vertices_add` ?

Reply:
Done